db_scripts/mosip_kernel/ddl/kernel-otp_transaction.sql

```diff
@@ -37,9 +37,9 @@ COMMENT ON TABLE kernel.otp_transaction IS 'OTP Transaction: All OTP related dat
 -- ddl-end --
 COMMENT ON COLUMN kernel.otp_transaction.id IS 'ID: Unique transaction id for each otp transaction request';
 -- ddl-end --
-COMMENT ON COLUMN kernel.otp_transaction.ref_id IS 'Reference ID: Reference ID is a reference information received from OTP requester which can be used while validating the OTP.';
```

Comment:
Example of ref_id: unique identifier for the OTP transaction

Reply:
ok